stories/useMutationsHandler/usemutationshandler.guideline.mdx

```diff
@@ -0,0 +1,156 @@
+import * as Stories from './usemutationshandler.stories';
+import { Meta, Controls, Story, Canvas, Source } from '@storybook/blocks';
+
+<Meta name="Guideline" of={Stories} />
+
+# useMutationsHandler
+
+useMutationsHandler is a hook to simplify the handling of mutations and the display of feedback to the user.
+It provide a way to define a callback to call on success and a toast to display after the fail or success of a mutation.
+It also allows to define a single toast and callback for multiple related mutations instead of managing each mutation individually.
```

Comment:
I suggest you another description : 

The `useMutationsHandler` hook is a powerful utility designed to manage and handle the results of multiple mutations using `react-query`. It provides a unified way to handle success and error states, display toast notifications, and manage side effects after mutation completions.